### PR TITLE
Updating version of mssql to recent

### PR DESCRIPTION
**Pull request by brett-kepler-walmart**
Updating version of mssql from 4.3.9 to 7.1.0 to try and debug issue that is happening with sql connections

### CODE REVIEW

**Comment by sklose** [comment]
can we push a commit here with [full ci] to ensure the integration tests all pass

**Comment by plameniv** [comment]
Brett mentioned that he successfully ran yarn integrate locally. It also looks like there is some connection issue that was the underlying cause of the problem he was seeing. We are looking at changing the timeout to see if that will fix the connection issues, so there may be additional changes on this PR. I will make sure we run [full ci] before the merge.

File: packages/mssql/src/index.ts [hunk 1]
```diff
@@ -71,6 +74,30 @@ class MssqlConfiguration {
...
+    @config.field(config.converters.boolean)
+    public set stream(_: boolean) {
+        config.noop();
+    }
+    public get stream(): boolean {
+        return config.noop();
+    }
 }
 
 export function mssqlSink(configuration: IMssqlConfiguration): IOutputSink<IPublishedMessage> {
```
Comment (plameniv) [inline packages/mssql/src/index.ts:103]: Do we want to set a default value for stream (or the timeout values), different from the one defined by mssql package? If so, we should add it here

File: packages/mssql/src/index.ts [hunk 2]
```diff
@@ -71,6 +74,30 @@ class MssqlConfiguration {
     public get mode(): Mode {
         return config.noop();
     }
+
+    @config.field(config.converters.number)
```
Comment (plameniv) [inline packages/mssql/src/index.ts:78]: Can you replace this with `@config.field(config.converters.timespan)`? It converts values into milliseconds, and that's what the unit that mssql accepts?

File: packages/mssql/src/index.ts (hunk 1 above)
Comment (brett-kepler-walmart) [reply packages/mssql/src/index.ts:103]: I don't think so because these changes will only matters in one place at the moment

**Review by plameniv — APPROVED** [review]

**Pull request merged**